app/models/mpc_offender.rb

```diff
@@ -0,0 +1,161 @@
+# frozen_string_literal: true
+
+class MpcOffender
```

Comment:
### Naming

It's a shame the `Offender` model name is already taken by our ActiveRecord model – since it seems like it would be a more suitable name for this model.

But I don't have a strong opinion about how better to name either of these models – so it doesn't seem worth changing!